control/controlclient/direct.go

```diff
@@ -328,6 +330,7 @@ func (c *Direct) doLogin(ctx context.Context, t *oauth2.Token, flags LoginFlags,
 
 	if expired {
 		c.logf("Old key expired -> regen=true")
+		systemd.Status("key expired, must regen")
```

Comment:
"regen" is a bit slang-y for user-visible errors. I'd say maybe:

    key expired; run 'tailscale up' to authenticate

?